src/reporter/reporter.py

```diff
@@ -147,15 +147,26 @@ def notify():
         fiware_sp = request.headers.get('fiware-servicepath', None)
     else:
         fiware_sp = None
-
-    # Send valid entities to translator
-    with translator_for(fiware_s) as trans:
-        trans.insert(payload, fiware_s, fiware_sp)
-
-    msg = 'Notification successfully processed'
+    for entity in payload:
+        attrs = list(iter_entity_attrs(entity))
+    attrs.pop()
+    length = (len(payload))
+    Flag = False
+    for i in range(length):
+        for j in attrs:
+            value = payload[i][j]['value']
+            if not value:
+                msg = 'Notification Not processed because of Null value'
+            else:
+                Flag = True
+                msg = 'Notification successfully processed'
+        if Flag:
```

Comment:
what if the payload contains three entities

    payload = e1, e2, e3

 `e1` and `e3` have values, `e2` does **not**. Isn't this sending the whole payload to the translator on the first loop iteration and then again on the third? Again sorry if there's something I'm missing, probably there is :-)

Reply:
@c0c0n3 , sorry i have missed that i have checked it for one entity. I am working on it.

Reply:
no worries, there's one gazillion things I miss too which is why we have code review :-)
thank you so much for working on this!